yaksh/templates/yaksh/question.html

```diff
@@ -184,6 +197,23 @@ <h4>Files to download for this question</h4>
                       <h5><a href="{{f_name.file.url}}">{{f_name.file.name}}</a></h5>
                       {% endfor %}
                       {% endif %}
+
+                        {% if quiz.is_exercise %}
+                            {% if question.solution %}
+                                <h4><u> Solution by teacher</u></h4>
```

Comment:
Solution by the teacher heading is shown by default, I feel it should be activated if the student gets the right answer or after the timeout.

Reply:
Done.
Actually causes slight redundancy so implemented like this.